app/controllers/concerns/verify_setup_completed.rb

```diff
@@ -0,0 +1,41 @@
+module VerifySetupCompleted
+  extend ActiveSupport::Concern
+
+  MANDATORY_CONFIGS = %i[
+    community_description
+    community_member_description
+    community_action
+    tag_url
+
+    main_social_image
+    favicon_url
+    logo_png
+    primary_sticker_image_url
```

Comment:
The `primary_sticker_image_url` config is strange to me, because it seems that we have a different logo for our internal admin interface (i.e. the primary_sticker_image_url which is the rainbowdev logo) and a different one for our external facing UI (i.e. the logo pg/svg). We also use  primary_sticker_image_url on the sign in invitation. 

I don't feel that every forem will have the luxury of having two logos and so my suggestion is as follows: 
- Let's make primary_sticker_image_url optional and without a default (so I'll take it out of this mandatory configs), and then we can use `logo_png` as the fallback when we do not have a primary_sticker_image_url. 
- In addition, perhaps we rename primary_sticker_image_url to `secondary_logo` since that is actually what our rainbowdev image is.

Any thoughts on this proposal?  (cc @ludwiczakpawel - would be good to have your thoughts here as well since it has a design/ui element)

Reply:
This sounds like a very sensible approach to me, the name `primary_sticker_image_url` is pretty confusing.

Reply:
@Ridhwana I definitely agree.

I believe most communities will have only one logo by default AND MAYBE another one for when they think it's for dark theme (which isn't valid assumption of course)... I don't think too many brands have "alternative" logo. So for whenever we reference any kind of logo in the UI - it should always be the same thing... TBH I can't really think of good usage for `secondary_logo` in the UI - there's no good reason to replace "normal" logo in certain places (like sign in invitation etc).

I'd personally nuke "secondary" logo altogether.

---

Another topic is that we need to provide a way for admins to setup other assets, like app icon, PWA icon (in multiple sizes too), favicon etc. etc.. Right now I think all of them are pulled from `/assets` folder in codebase but maybe this could be configurable from admin panel as well?